### PR TITLE
Fix double padding when using ContainedGrid and PageSection

### PR DESCRIPTION
**Pull request by sarunast**
If the `container` classes gets nested it will multiply the padding on left and right. This is especially visible on small devices.

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit db92b71a10f579630c3c7b7c91772517bb1fef56

https://deploy-preview-216--masifunde.netlify.com

File: components/ContainedGrid.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 import PropTypes from 'prop-types'
 
-const ContainedGrid = ({ children }) => (
-  <div className="container">
+const ContainedGrid = ({
+  children,
+  className,
+  withoutContainer,
+}) => (
+  <div className={`${className} ${withoutContainer ? '' : 'container'}`}>
```
Comment (erikjalevik) [inline components/ContainedGrid.js]: Isn't the whole purpose of something called `ContainedGrid` to add a container?

Comment (sarunast) [reply components/ContainedGrid.js]: Yeah, I was thinking about the naming. But I couldn't come up with something reasonable.

Comment (sarunast) [reply components/ContainedGrid.js]: I will rename it to `CenteredGrid`

**Pull request merged**